lib/rubocop/cop/rails/timecop.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop disallows all usage of `Timecop`, in favour of
+      # `ActiveSupport::Testing::TimeHelpers`.
+      #
+      # ## Migration
+      # `Timecop.freeze` should be replaced with `freeze_time` when used
+      # without arguments. Where a `duration` has been passed to `freeze`, it
+      # should be replaced with `travel`. Likewise, where a `time` has been
+      # passed to `freeze`, it should be replaced with `travel_to`.
```

Comment:
`TimeHelper` always freezes time. In fact, `freeze_time` is implemented by calling `travel_to` with `Time.now`. I believe they took the approach of having fewer features to force tests to be clear and explicit.

This means `Timecop.freeze(duration)` and `Timecop.freeze(time)` can safely be replaced with `travel(duration)` and `travel_to(time)` respectively, without needing to call `freeze_time`.

We could _try_ to analyze the arguments to check for simple cases matching the `<number>.<unit>` or `<number>.<unit>.<relative direction>` patterns and correct accordingly, but this would add a bunch of complexity. I'd be inclined to do this in a separate PR.

Reply:
>  In fact, freeze_time is implemented by calling travel_to with Time.now ...

Yikes, that's a big change in the API semantics going from Timecop to TimeHelper. (If I understand correctly.) Makes me even more convinced that this cop should not be enabled by default.